### PR TITLE
added sean to people.yml file

### PR DESCRIPTION
**Pull request by SeanWu25**
I just added myself to people.yml. Sorry for the delay.

### CODE REVIEW

File: db/people.yml [hunk 1]
```diff
@@ -5339,8 +5339,11 @@ swu:
...
   aka: []
   avatar: "https://github.com/\x16SeanWu25.png"
-  bio: bio
-  education: []
+  bio: Sean is a third year undergraduate student at Pepperdine University. He is
+    studying Computer Science/Mathematics and is interested in advancing medicine with machine learning.
+    He loves to play basketball and piano. He is conducting research with the Columbia SURE
+    program for Summer 2023.
+  education: [Pepperdine]
   email: "\x16sean.wu@pepperdine.edu"
```
Comment (sbillinge) [inline db/people.yml:5347]: remove \x16

File: db/people.yml [hunk 2]
```diff
@@ -5339,8 +5339,11 @@ swu:
   active: true
   aka: []
   avatar: "https://github.com/\x16SeanWu25.png"
-  bio: bio
-  education: []
+  bio: Sean is a third year undergraduate student at Pepperdine University. He is
+    studying Computer Science/Mathematics and is interested in advancing medicine with machine learning.
+    He loves to play basketball and piano. He is conducting research with the Columbia SURE
+    program for Summer 2023.
+  education: [Pepperdine]
```
Comment (sbillinge) [inline db/people.yml:5346]: see other people's entry to understand how to do this

File: db/people.yml [hunk 3]
```diff
@@ -5339,8 +5339,11 @@ swu:
   active: true
   aka: []
   avatar: "https://github.com/\x16SeanWu25.png"
-  bio: bio
-  education: []
+  bio: Sean is a third year undergraduate student at Pepperdine University. He is
+    studying Computer Science/Mathematics and is interested in advancing medicine with machine learning.
```
Comment (sbillinge) [inline db/people.yml:5343]: there may be intendation errors here due to line wrapping too, not sure

**Review by sbillinge — COMMENTED** [review]
please see inline comments.

Also, when you make a PR you can check if it is passing CI (continuous integration).  Go to the PR (conversation tab) and look for this:
image
then click on details and you can see what is the problem with your PR

**Comment by sbillinge** [comment]
@SeanWu25 can we try to work to merge this?  It should be quick now you have a better idea about things.

**Comment by SeanWu25** [comment]
Yes I am working on it right now

File: db/people.yml [hunk 4]
```diff
@@ -5338,26 +5338,23 @@ swhan:
...
   aka: []
-  avatar: "https://github.com/\x16SeanWu25.png"
-  bio: Sean is a third year undergraduate student at Pepperdine University. He is
-    studying Computer Science/Mathematics and is interested in advancing medicine with machine learning.
-    He loves to play basketball and piano. He is conducting research with the Columbia SURE
-    program for Summer 2023.
-  education: [Pepperdine]
-  email: "\x16sean.wu@pepperdine.edu"
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: 'Sean is a third year undergraduate student at Pepperdine University. He is studying Computer Science/Mathematics and is interested in advancing medicine with machine learning. He loves to play basketball and piano. He is conducting research with the Columbia SURE program for Summer 2023.'
```
Comment (sbillinge) [inline db/people.yml:5342]: please fix indentation

File: db/people.yml [hunk 5]
```diff
@@ -5338,26 +5338,23 @@ swhan:
...
-      group: bg
+    - begin_year: 2023
+      end_year: 2023
+      end_month: 8
+      begin_month: 5
+      position: visiting student
+      organization: pepperdine university
       advisor: sbillinge
-      status: undergrad
-  github_id: "\x16SeanWu25"
```
Comment (sbillinge) [inline db/people.yml:5353]: this is required, please add back, we just need to remove the rogue `\x16`

File: db/people.yml [hunk 6]
```diff
@@ -5338,26 +5338,23 @@ swhan:
...
-  grp_mtg_active: true
```
Comment (sbillinge) [inline db/people.yml:5354]: required, add back

File: db/people.yml [hunk 7]
```diff
@@ -5338,26 +5338,23 @@ swhan:
...
+      group: bg
   name: Sean Wu
-  orcid_id: ''
-  position: undergraduate researcher
+  position: Undergradute Visitor
```
Comment (sbillinge) [inline db/people.yml:5357]: please restore these two fields

File: db/people.yml [hunk 8]
```diff
@@ -5338,26 +5338,23 @@ swhan:
...
-    He loves to play basketball and piano. He is conducting research with the Columbia SURE
-    program for Summer 2023.
-  education: [Pepperdine]
-  email: "\x16sean.wu@pepperdine.edu"
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: 'Sean is a third year undergraduate student at Pepperdine University. He is studying Computer Science/Mathematics and is interested in advancing medicine with machine learning. He loves to play basketball and piano. He is conducting research with the Columbia SURE program for Summer 2023.'
+  education:
+    - begin_year: 2023
+      institution: pepperdine university
+      degree: visiting student
```
Comment (sbillinge) [inline db/people.yml:5346]: probably your degree is something like BA, BSc something ike that.  Please look at other entries to get the right format.

Please add begin_month and you can put end_month and end_year in the future.

**Review by sbillinge — COMMENTED** [review]
please see inline comments

File: db/people.yml (hunk 8 above)
Comment (SeanWu25) [reply db/people.yml:5346]: On it right now

File: db/people.yml (hunk 7 above)
Comment (SeanWu25) [reply db/people.yml:5357]: On it right now

File: db/people.yml (hunk 6 above)
Comment (SeanWu25) [reply db/people.yml:5354]: On it right now

File: db/people.yml (hunk 5 above)
Comment (SeanWu25) [reply db/people.yml:5353]: On it right now

**Pull request merged**